docs/specs/sections/definitions.tex

```diff
@@ -0,0 +1,99 @@
+\subsection{The Roles involved} 
+
+\begin{itemize}
+    \item \textbf{User:} An entity that interacts with the wallet to request licenses and prove ownership of those.
+    \item \textbf{Service Provider (SP):} An entity offering an off-chain service that receives requests for licenses, and upon acceptance, issues them.
+    \item \textbf{Session Service Provider (SSP):} It provides the service upon verification that a service request is correct. It can be the same individual that the SP or another one.
+\end{itemize}
+
+\subsection{The Elements involved} 
+
+\begin{itemize}
+    \item \textbf{Request:} A request includes the encryption of a stealth address belonging to the user, where the license has to be sent to, and a symmetric key. The structure is as follows:
+
+    \begin{center}
+        \begin{tabular}{ |c|c|c|c| } 
+        \hline
+        \textbf{Element} & \textbf{Type} & \textbf{Info.} \\
+        \hline
+        $(\rpk, R)$ & StealthAddress & It is a request stealth address of the SP. \\
+        $\enc$ & PoseidonCipher & It is a ciphertext of size 6. \\
+        $\nonce$ & BlsScalar & It is a randomness needed to compute $\enc$. \\ 
+        \hline
+        \end{tabular}
+    \end{center}
+
+
+    \item \textbf{License:} A license is an asset that represents the right of a user to use a given service. The structure is as follows:
+
+    \begin{center}
+        \begin{tabular}{ |c|c|c|c| } 
+        \hline
+        \textbf{Element} & \textbf{Type} & \textbf{Info.} \\
+        \hline
+        $(\lpk, R)$ & StealthAddress & It is a license stealth address of the user. \\
```

Comment:
yes

Reply:
Does `spend` mean something idempotent, like proving ownership only, or it is the act of final nullification? Because the word `spend` intuitively suggests that after that the license is `spent`, no longer usable.

Reply:
that's the case. You usually spend.... an """"instantiation""""" of a license. If it can be used many times, you will be able to instatiate it multiple times.